libgadget/tests/test_forcetree.c

```diff
@@ -47,6 +47,10 @@ struct forcetree_testdata
 
 void dump_snapshot() { }
 
+double walltime_measure_full(char * name, char * file, int line) {
+    return MPI_Wtime();
+}
+
 /*End dummies*/
```

Comment:
I recently learned the proper name of these dummies are mocks.

Reply:
Well sure but as a british chap I delight in using unusual arguably incorrect names for things :-)

Reply:
I don't actually know the origin of Mock. Dummy actually reads more lively.